scalafmt-core/shared/src/main/scala/org/scalafmt/internal/FormatOps.scala

```diff
@@ -2113,7 +2117,16 @@ class FormatOps(
               if !hasSingleTermStat(t) && isBlockStart(t, nft) =>
             Some(new OptionalBracesRegion {
               def owner = t.parent
-              def splits = Some(getSplitsMaybeBlock(ft, nft, t, true))
+              def splits = Some(
+                getSplitsMaybeBlock(
+                  ft,
+                  nft,
+                  t,
+                  true,
```

Comment:
since you are already unfolding the parameters, could you explicitly name this boolean, too,

Reply:
Named, and also named the other one.